<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeChooser</class>
 <widget class="QWidget" name="ModeChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор режима</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="projectComparison">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="metodicJudges">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эксперты методики</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sectionJudges">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эксперты разделов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rangedConstantsJudges1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эксперты ранжируемых факторов первого проекта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rangedConstantsJudges2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эксперты ранжируемых факторов второго проекта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sourceData">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Исходные данные проектов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sectionsCalculation">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Расчёт разделов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculation">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Расчёт проектов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="help">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Справка</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="exit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>ModeChooser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>785</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
